src/script/stores/uiStore.ts

```diff
@@ -20,7 +20,7 @@ import { DeviceRequestStates } from '../microbit-interfacing/MicrobitConnection'
 let text: (key: string, vars?: object) => string;
 t.subscribe(t => (text = t));
 
-export const compatibility = writable<CompatibilityStatus>(checkCompatibility());
+export const compatibility = writable<CompatibilityStatus>(await checkCompatibility());
```

Comment:
How worried should I be that this has regressed browser compatibility by using top-level await?

Reply:
Very broken on Chrome 88: 
`Uncaught SyntaxError: Unexpected reserved word`

Reply:
Making this a draft until resolved.

Reply:
We have another one in i18n.ts:

```
await init({
  fallbackLocale: 'en',
  initialLocale: get(persistantLocale),
  // Needed to format <link> style tags.
  ignoreTag: false,
});
```

Reply:
This is already broken for Chrome 88 due to the await in i18n.ts. We'll come back to this later to attempt to target es2017 and lose both top-level awaits.